media/CircleGraph/index.js

```diff
@@ -492,7 +492,7 @@ host.BrowserHost = class {
         this._view.setSelection(message);
     }
 
-    _msgColorTheme(message) {
+    _msgColorTheme(_message) {
```

Comment:
I could not fully understand why not to remove the argument. 😢 As the title of this pr, both `message` isn't used in the method. 
@seanshpark @dayo09 Was there any background to remain the argument as variable but adding underscore in front of the name?

Reply:
because I wanted to preserve initial design of using message argument. currently I don't have time to take another look with other codes it would be appropriate to remove or not.

adding underscore will make ignore the tool used for unused variable.

Reply:
@yunjayh Well I think so too. :sweat_smile:  I just put the author's mind as the top priority. I introduced the underscore-ignoring rules to make a room for it.